ui/apps/platform/src/Containers/VulnMgmt/Reports/Form/NotifierSelection.tsx

```diff
@@ -20,10 +20,10 @@ import EmailNotifierFormModal from './EmailNotifierFormModal';
 type NotifierSelectionProps = {
     notifierId: string;
     mailingLists: string[];
-    setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void;
-    handleBlur: (e: React.FocusEvent<any, Element>) => void;
-    errors: FormikErrors<any>;
-    touched: FormikTouched<any>;
+    setFieldValue: (field: string, value: unknown, shouldValidate?: boolean | undefined) => void;
+    handleBlur: (e: React.FocusEvent<unknown, Element>) => void;
+    errors: FormikErrors<unknown>;
+    touched: FormikTouched<unknown>;
```

Comment:
are you able to use `ReportFormValues` here and on line 25?

Reply:
Good question.
* It looks like `ReportFormValues` is in Vulnerability Management Reports 2.0
* Too bad, so sad, this is in Vulnerability Management 1.0 and not even in bundle